app/build.gradle

```diff
@@ -49,6 +48,7 @@ android {
         abortOnError false
         disable 'InvalidPackage'
     }
+    buildToolsVersion '27.0.3'
```

Comment:
This should not be changed here

Reply:
@iamareebjamal As each time this requires to be updated, should I open a new issue to change it?

Reply:
Yes, open a new issue to upgrade all dependencies